pkg/reloader/reloader.go

```diff
@@ -114,6 +118,20 @@ type TriggerReloader interface {
 	TriggerReload(ctx context.Context) error
 }
 
+// SubConfigOptions contains options for a sub-config. For example, these could be something
+// such as additional scrape config files or rule files listed in the main Prometheus
+// configuration. Sub-directories are ignored.
+type SubConfigOptions struct {
```

Comment:
```suggestion
type CfgOption struct {
```

or `CfgDirOption`? (see other comment why)